apps/sasquatch/src/main/java/com/microsoft/appcenter/sasquatch/activities/EventActivity.java

```diff
@@ -103,6 +110,41 @@ public void onClick(View v) {
                 startActivity(intent);
             }
         });
+
+        /* Init pause/resume buttons. */
+        /* TODO remove reflection once SDKs are released. */
+        mPauseTransmissionButton = findViewById(R.id.pause_transmission_button);
+        mPauseTransmissionButton.setOnClickListener(new View.OnClickListener() {
+
+            @Override
+            public void onClick(View v) {
+                AnalyticsTransmissionTarget target = getSelectedTarget();
+                try {
+                    final Method pauseMethod = target.getClass().getMethod("pause");
```

Comment:
In java using final variables in methods is very unusual unless it is used into an inner anonymous class which is not the case here.